### ISSUE TITLE
example of getting the consensus of multiple sequences?

### ISSUE THREAD

**Issue by dongweigogo** [biogo/biogo#73]
I have many similar sequences of different lengths and need to get the consensus of them.
From the docs, the seq module should work for this, but any example provided?

**Comment by dongweigogo** [comment]
I've found exmaples in the codes. Thanks.

**Comment by dongweigogo** [comment]
After reading the docs, I tried the multi type for getting the consensus of multiple similar sequences, but it just did not produce what I want. Did I missed something? 
My code is as below:

`

        m, err := multi.NewMulti("test",
                []seq.Sequence{
                        linear.NewSeq("seq1", []alphabet.Letter("AAAATTTT"), alphabet.DNA),
                        linear.NewSeq("seq2", []alphabet.Letter("GGGAAAAT"), alphabet.DNA),
                        linear.NewSeq("seq3", []alphabet.Letter("ATTTTCCC"), alphabet.DNA),
                },
                seq.DefaultConsensus)
        if err != nil {
                fmt.Println(err)
                return
        }
        fmt.Printf("%- s\n\n%-s\n\n\n", m, m.Consensus(false))

`

it gives:
AAAATTTT
GGGAAAAT
ATTTTCCC

nnnannnt

but what I want is something like this:
seq1: 　　AAAATTTT
seq2: GGGAAAAT
seq3: 　　　　ATTTTCCC

cons: GGGAAAATTTTCCC

**Comment by kortschak** [comment]
If you want an alignment, you need to provide the alignment. The concensus of a multiseq is purely that, just the consensus of the sequences that you give it. For something that does all of this, see [this code](https://github.com/biogo/examples/blob/b7a3754bac76d55b005ebd39829b13c5be039be3/igor/seqer/seqer.go#L301-L335)

**Comment by dongweigogo** [comment]
It seems that the code relies on external mafft/muscle for multiple alignment, is biogo able to do this by itself?

**Comment by kortschak** [comment]
No. Multiple sequence alignment is non-trivial and it doesn't seem like a worthwhile investment of time given that there are good tools that already do it.

**Comment by dongweigogo** [comment]
I see. Thanks.
